extensions/wikia/Rail/templates/RailController_Index.php

```diff
@@ -1,25 +1,27 @@
 <? if ( $isAside ): ?>
 	<aside>
 <? endif; ?>
-	<div id="WikiaRail" class="WikiaRail<?= !empty($isGridLayoutEnabled) ? ' grid-2' : '' ?>">
+	<div id="WikiaRailWrapper" class="WikiaRail<?= !empty($isGridLayoutEnabled) ? ' grid-2' : '' ?>">
+		<div id="WikiaRail">
 
-		<?php
-		// sort in reverse order (highest priority displays first)
-		krsort($railModuleList);
+			<?php
+			// sort in reverse order (highest priority displays first)
+			krsort($railModuleList);
 
-		// render all our rail modules here
-		foreach ($railModuleList as $priority => $callSpec) {
-			echo F::app()->renderView(
-				$callSpec[0], // controller
-				$callSpec[1], // method
-				$callSpec[2]  // method's params
-			);
-		}
-		?>
-		<? if ($loadLazyRail): ?>
-			<div class="loading"></div>
-		<? endif ?>
+			// render all our rail modules here
+			foreach ($railModuleList as $priority => $callSpec) {
+				echo F::app()->renderView(
+					$callSpec[0], // controller
+					$callSpec[1], // method
+					$callSpec[2]  // method's params
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)